modules/backend/clinic/staff/room/i_room.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+
+class iRoom
+{
+  public:
+    virtual ~iRoom() = 0;
```

Comment:
We need to consider for each class if can extract and reasonable base interface. If not, we should  decide which level should be BaseClass.